### DISCUSSION TITLE
Missing projects view

### DISCUSSION THREAD

**Discussion by PatrykPospiech** [community/community#5789] [Mobile] [answered]
Is it possible to add feature that it will be possible to open and edit projects not related to any repository ? 

the view with i talking about is GitHub.com/PatrykPospiech?tab=projects on the website 

Thanks, for your time :)

**Comment by brianlovin** [comment, marked as answer]
Thanks for the suggestion @PatrykPospiech! I've added your idea to our backlog; we don't have plans to build this in the short term, but we'll keep it in mind as we work on profiles and projects in the future.
